presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4697,6 +4697,13 @@ public void testAccessControl()
         assertAccessDenied("SELECT name AS my_alias FROM nation", "Cannot select from columns \\[name\\] in table .*.nation.*", privilege("name", SELECT_COLUMN));
         assertAccessDenied("SHOW CREATE TABLE orders", "Cannot show columns of table .*.orders.*", privilege("orders", SHOW_COLUMNS));
         assertAccessAllowed("SHOW CREATE TABLE lineitem", privilege("orders", SHOW_COLUMNS));
+        assertAccessAllowed("SHOW STATS FOR lineitem");
+        assertAccessAllowed("SHOW STATS FOR lineitem", privilege("orders", SELECT_COLUMN));
+        assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)");
+        assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)", privilege("orders", SELECT_COLUMN));
+        assertAccessDenied("SHOW STATS FOR (SELECT * FROM lineitem)", "Cannot select from columns \\[.*\\] in table or view .*.lineitem.*", privilege("lineitem", SELECT_COLUMN));
+        assertAccessDenied("SHOW STATS FOR (SELECT nationkey FROM nation)", "Cannot select from columns \\[.*\\] in table or view .*.nation.*", privilege("nation", SELECT_COLUMN));
```

Comment:
provide exact columns in the message

here it's only one, but would be good in other cases too